assets/out

```diff
@@ -74,7 +74,7 @@ load_base=$(jq -r '.params.load_base // ""' < $payload)
 build=$(jq -r '.params.build // ""' < $payload)
 cache=$(jq -r '.params.cache' < $payload)
 cache_tag=$(jq -r ".params.cache_tag // \"${tag_name}\"" < $payload)
-dockerfile=$(jq -r '.params.dockerfile // ""' < $payload)
+dockerfile=$(jq -r ".params.dockerfile // \"${build}/Dockerfile\"" < $payload)
```

Comment:
It may be worthwhile however to in general introduce a

```bash
if [ ! -f $dockerfile ]; then
   echo "Dockerfile doesn't appear to exist: $dockerfile"
fi
``` 

cause if it doesn't (e.g. if/when I used the `dockerfile:` key but specified the parent directory out of habit) the `grep` I added caused the process to silently fail